var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py

```diff
@@ -27,12 +27,42 @@ class GcpMetadata(BasicModifier):
     maintainers("rfbgo")
 
     mode("standard", description="Standard execution mode")
+    mode(
+        "local", description="Local execution (disables parallel prefix/pdssh)"
+    )
     default_mode("standard")
 
     software_spec("pdsh", pkg_spec="pdsh", package_manager="spack*")
 
     required_variable("hostlist")
 
+    modifier_variable(
+        "metadata_parallel_prefix",
+        default="pdsh -R ssh -N -w {hostlist} '",
+        modes=["standard"],
+        description="Express how parlalelism should be done between nodes",
+    )
+    modifier_variable(
+        "metadata_parallel_prefix",
```

Comment:
Pretty nit-picky, but should this be called `metadata_prefix` instead of `metadata_parallel_prefix`, since we attempt to ignore it in serial anyway, but it could technically be used to pre/suffix commands when running locally too.

Reply:
Yea I had a hard time naming it. I did go with `metadata_prefix` to start but I think that is actually very misleading, so it only affect those commands marked with `per_node=True` (hence the name being my best attempt to convey that).

I think if we went with  `metadata_prefix` people would be pretty surprised with how it acts when setting it from the YAML.